bioblend/galaxy/invocations/__init__.py

```diff
@@ -160,14 +160,48 @@ def get_invocation_summary(self, invocation_id):
         url = self._make_url(invocation_id) + '/jobs_summary'
         return self._get(url=url)
 
-    def get_invocation_report(self, invocation_id):
+    def get_invocation_step_jobs_summary(self, invocation_id):
         """
-        Get a Markdown report for an invocation.
+        Get a summary of an invocation, stating the number of jobs which
+        succeed, which are paused and which have errored.
 
         :type invocation_id: str
         :param invocation_id: Encoded workflow invocation ID
 
-        :rtype: dict
+        :rtype: dict[]
+        :return: The invocation step jobs summary.
+          For example::
+            [
+                {'populated_state': 'ok',
+                 'states': {'ok': 1},
+                 'model': 'Job',
+                 'id': 'e85a3be143d5905b'},
+
+                {'populated_state': 'ok',
+                 'states': {'running': 1},
+                 'model': 'Job',
+                 'id': 'c9468fdb6dc5c5f1'}
+
+                {'populated_state': 'ok',
+                 'states': {'new': 1},
+                 'model': 'Job',
+                 'id': '2a56795cad3c7db3'}
+            ]
+        """
+        url = self._make_url(invocation_id) + '/step_jobs_summary'
+        return self._get(url=url)
+
+    def get_invocation_report(self, invocation_id, format='markdown'):
```

Comment:
Should we require an output file object as argument to save the PDF report, similarly to https://github.com/galaxyproject/bioblend/blob/main/bioblend/galaxy/histories/__init__.py#L652-L679 ?
In that case, maybe keeping the PDF report method separate could be better @simonbray ?

Reply:
Yeah, I just realised that the PDF should be saved as a output file... sorry @rikeshi, can you revert back to the old version with a separate method, and include a `file_path` argument?

Reply:
Okay

Reply:
```suggestion
    def get_invocation_report(self, invocation_id):
```